content/ecosystem.md

```diff
@@ -0,0 +1,21 @@
+
+
+## Open Source Projects
+
+-   ##### node-red-contrib-wazo-platform
+    This nodes permit to connect Wazo Platform to [Node-RED](https://nodered.org/):[node-red-contrib-wazo-platform](https://flows.nodered.org/node/node-red-contrib-wazo-platform)
+
+
+
+## Commercial Products
+
+-   ##### Canyan
```

Comment:
@tranchitella @alexsosic Are you using Wazo Platform in Cnyan? I don't see a reference anymore on the website.

Reply:
@fredericlepied No, we're writing our own code for the SBC and Router. :)

Reply:
@fredericlepied yes, we plan to use Wazo Platform in Canyan. We are in the process of updating the website, though, that's why you don't see a reference to wazo-platform.

Reply:
@alexsosic ok removing the reference to Canyan.